src/main/java/com/thealgorithms/datastructures/graphs/BipartiteGraphDFS.java

```diff
@@ -35,11 +45,17 @@ private static boolean bipartite(int v, ArrayList<ArrayList<Integer>> adj, int[]
         return true;
     }
 
+    /**
+     * Method to check if the graph is bipartite.
+     *
+     * @param v   Number of vertices in the graph
+     * @param adj Adjacency list of the graph
+     * @return    True if the graph is bipartite, otherwise false
+     */
     public static boolean isBipartite(int v, ArrayList<ArrayList<Integer>> adj) {
         // Code here
```

Comment:
This is not from this PR, but can be removed.

Reply:
Done!